### ISSUE TITLE
 Console.log in main process not logged to terminal in 3.0.0-alpha.1 release

### ISSUE THREAD

**Issue by steveswinbank** [nklayman/vue-cli-plugin-electron-builder#1641]
**Describe the bug**
console.log() logging is not sent to terminal from the main process. This is different behavior to version 2.1.1

**To Reproduce**
- install new alpha version following instructions here: [https://bestofvue.com/repo/nklayman-vue-cli-plugin-electron-builder-vuejs-vue-cli-3-plugins#releases](url)
- add a console.log('some message...') logging statement in background.js
- run using `npm run electron:serve`
- observe that no further logging to terminal beyond  "INFO  Launching Electron..."

**Expected behavior**
- expect message 'some message...' to be logged to the terminal

**Environment (please complete the following information):**
 - terminal output from running `vue info`:
 ```
 - System:
    OS: macOS 12.1
    CPU: (10) arm64 Apple M1 Pro
  Binaries:
    Node: 16.13.1 - /opt/homebrew/bin/node
    Yarn: 1.22.17 - /opt/homebrew/bin/yarn
    npm: 8.1.2 - /opt/homebrew/bin/npm
  Browsers:
    Chrome: 96.0.4664.110
    Edge: Not Found
    Firefox: 95.0.2
    Safari: 15.2
  npmPackages:
    @vue/babel-helper-vue-jsx-merge-props:  1.2.1 
    @vue/babel-helper-vue-transform-on:  1.0.2 
    @vue/babel-plugin-jsx:  1.1.1 
    @vue/babel-plugin-transform-vue-jsx:  1.2.1 
    @vue/babel-preset-app:  5.0.0-rc.1 
    @vue/babel-preset-jsx:  1.2.4 
    @vue/babel-sugar-composition-api-inject-h:  1.2.1 
    @vue/babel-sugar-composition-api-render-instance:  1.2.4 
    @vue/babel-sugar-functional-vue:  1.2.2 
    @vue/babel-sugar-inject-h:  1.2.2 
    @vue/babel-sugar-v-model:  1.2.3 
    @vue/babel-sugar-v-on:  1.2.3 
    @vue/cli-overlay:  5.0.0-rc.1 
    @vue/cli-plugin-babel: ~5.0.0-rc.1 => 5.0.0-rc.1 
    @vue/cli-plugin-eslint: ~5.0.0-rc.1 => 5.0.0-rc.1 
    @vue/cli-plugin-pwa: ~5.0.0-rc.1 => 5.0.0-rc.1 
    @vue/cli-plugin-router: ~5.0.0-rc.1 => 5.0.0-rc.1 
    @vue/cli-plugin-unit-jest: ~5.0.0-rc.1 => 5.0.0-rc.1 
    @vue/cli-plugin-vuex: ~5.0.0-rc.1 => 5.0.0-rc.1 
    @vue/cli-service: ~5.0.0-rc.1 => 5.0.0-rc.1 
    @vue/cli-shared-utils:  5.0.0-rc.1 
    @vue/component-compiler-utils:  3.3.0 
    @vue/eslint-config-airbnb: ^6.0.0 => 6.0.0 
    @vue/test-utils: ^1.1.3 => 1.3.0 
    @vue/vue2-jest: ^27.0.0-alpha.2 => 27.0.0-alpha.4 
    @vue/web-component-wrapper:  1.3.0 
    eslint-plugin-vue: ^8.0.3 => 8.2.0 
    eslint-plugin-vuejs-accessibility: ^1.1.0 => 1.1.1 
    jest-serializer-vue:  2.0.2 
    vue: ^2.6.14 => 2.6.14 
    vue-cli-plugin-electron-builder: ^3.0.0-alpha.1 => 3.0.0-alpha.1 
    vue-eslint-parser:  8.0.1 
    vue-hot-reload-api:  2.3.4 
    vue-loader:  16.8.3 (15.9.8)
    vue-router: ^3.5.1 => 3.5.3 
    vue-style-loader:  4.1.3 
    vue-template-compiler: ^2.6.14 => 2.6.14 
    vue-template-es2015-compiler:  1.9.1 
    vuex: ^3.6.2 => 3.6.2 
  npmGlobalPackages:
    @vue/cli: 5.0.0-rc.1
```

**Comment by nklayman** [comment]
Just realized this while debugging another issue, thanks for confirming it. Just published `v3.0.0-alpha.3` with a fix, let me know if it works for you.

**Comment by steveswinbank** [comment]
Thanks for the update. I can confirm `console.log()` is working in version `v3.0.0-alpha.3`
